nss-day-cohort-48/giffygram-jumpstation
# Smb recieving messages

#### Changes Made

1. Modified file `NavBar.js` to include `Full nav functionality` and functionality.
2. Modified DirectMessage to display received messages. 
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout branchname
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Make sure the nav bar icons all render the page properly. The message form sends the message to your database when save is clicked. 
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
